components/emoji_picker/emoji_picker.test.jsx

```diff
@@ -132,14 +132,12 @@ describe('components/emoji_picker/EmojiPicker', () => {
         expect(wrapper.find('.emoji-picker__items').prop('style')).toStrictEqual({overflowY: 'hidden'});
         expect(wrapper.find(EmojiPickerCategorySection).length).toBe(2);
 
-        jest.runOnlyPendingTimers();
-        wrapper.update();
-        jest.runOnlyPendingTimers();
+        setTimeout(() => {
```

Comment:
i'm not really liking this change, but seems that now initial perf is worse by having a two categories first and then moving into having all of them due to only having 1 bigger spritesheet instead of multiple smaller ones.